packages/rocketchat-ui-sidenav/client/accountBox.html

```diff
@@ -6,7 +6,11 @@
 					{{> avatar username=username}}
 				</div>
 				<div class="sidebar__account-data">
-					<h3 class="sidebar__account-username" data-username="{{username}}">@{{username}}</h3>
+					{{#if fname}}
```

Comment:
I think we should use a helper here instead an if and the same html.. what do you think @geekgonecrazy ?